### PR TITLE
bugfix/1079 Extensions Longer Than 128 Chars

### PR DESCRIPTION
**Pull request by ggrg**
- truncateExtension config
- fix issues with standardJS

### CODE REVIEW

**Comment by KamuelaFranco** [comment]
@ggrg Thanks for this. Can you summarize the functionality that changes as a result of this PR?

**Comment by rmothilal** [comment]
> @ggrg Thanks for this. Can you summarize the functionality that changes as a result of this PR?

@KamuelaFranco This is to truncate the extensions for error objects being created in the toApiErrorObject function call

**Comment by elnyry** [comment]
> @ggrg Thanks for this. Can you summarize the functionality that changes as a result of this PR?

@KamuelaFranco please refer to the issue mojaloop/project/issues/1079 for more details along with the 'description section' above

These may be helpful as well
- https://github.com/mojaloop/central-services-error-handling/pull/91
- https://github.com/mojaloop/central-services-error-handling/pull/92
- https://github.com/mojaloop/central-services-shared/pull/134
